orchagent/vnetorch.cpp

```diff
@@ -551,11 +553,45 @@ bool VNetOrch::delOperation(const Request& request)
     return true;
 }
 
+bool VNetOrch::getVrfIdByVnetName(const std::string& vnet_name, sai_object_id_t &vrf_id)
+{
+    if (!isVnetExists(vnet_name))
+    {
+        return false;
+    }
+
+    auto *vrf_obj = getTypePtr<VNetVrfObject>(vnet_name);
+    // Now we only support ingress VR for VNET, so just get ingress VR ID
+    // Once we support egress VR, need revisit here.
+    vrf_id = vrf_obj->getVRidIngress();
+    return vrf_id != SAI_NULL_OBJECT_ID;
+}
+
+bool VNetOrch::getVnetNameByVrfId(sai_object_id_t vrf_id, std::string& vnet_name)
+{
+    for (auto &entry : vnet_table_)
+    {
+        auto *vrf_obj = dynamic_cast<VNetVrfObject *>(entry.second.get());
+        if (!vrf_obj)
+        {
+            continue;
+        }
+
+        if (vrf_obj->getVRidIngress() == vrf_id)
+        {
+            vnet_name = entry.first;
+            return true;
+        }
+    }
+
+    return false;
+}
+
 /*
  * Vnet Route Handling
  */
 
-static bool del_route(sai_object_id_t vr_id, sai_ip_prefix_t& ip_pfx)
+static bool del_route(sai_object_id_t vr_id, sai_ip_prefix_t& ip_pfx, const IpPrefix& ipPrefix)
```

Comment:
This is too many changes in API definitions. Cherry-picking a future fix will be causing conflicts. Can you derive ipPrefix from ip_pfx?